README.md

```diff
@@ -67,6 +67,29 @@ cfg.dump("path_to_dump_dir", "object_id")
 equals_cfg = model_agnostic_load("path_to_dump_dir", "object_id", models=[MyNumpyModel, MyDemoModel])
 ```
 
+### Custom serialization
+
+If you're not happy with default `NumpyDataDict` serialization, as defined in [typing.py](./pydantic_numpy/helper/typing.py), you could define your own type with custom serializer using `NpArrayPydanticAnnotation.factory`
+and its `serialize_numpy_array_to_json` argument taking `Callable[[npt.ArrayLike], Iterable]`.
```

Comment:
Maybe something like this?

```suggestion
If the default serialization of NumpyDataDict, as outlined in [typing.py](https://chat.openai.com/c/pydantic_numpy/helper/typing.py), doesn't meet your requirements, you have the option to define a custom type with its own serializer. This can be achieved using the NpArrayPydanticAnnotation.factory method, which accepts a custom serialization function through its serialize_numpy_array_to_json parameter. This parameter expects a function of the form Callable[[npt.ArrayLike], Iterable], allowing you to tailor the serialization process to your specific needs.
```

Reply:
No comment :laughing: